cmd/kubeadm/app/util/config/cluster.go

```diff
@@ -29,6 +29,7 @@ import (
 )
 
 // FetchConfigFromFileOrCluster fetches configuration required for upgrading your cluster from a file (which has precedence) or a ConfigMap in the cluster
+// TODO: This func should be renamed FetchClusterConfigFromFileOrCluster, and return a ClusterConfiguration instead of an InitConfiguration
```

Comment:
Please add TODO: Rename constant `constants.InitConfigurationConfigMap` to `constants.ClusterConfigurationConfigMap`